src/telliot_feeds/sources/numeric_api_response.py

```diff
@@ -82,7 +82,9 @@ async def fetch_new_datapoint(self) -> OptionalDataPoint[Any]:
 
         datapoint = (val, datetime_now_utc())
         self.store_datapoint(datapoint)
-
+        print(f"Response value {datapoint[0]} parsed using parse strings [{self.parseStr}] will be submitted on chain;")
```

Comment:
I don't think this one should have any blocking logic, no? Isn't this un supposed to be automatic?